app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationTopAppBar.kt

```diff
@@ -194,6 +180,24 @@ private fun ConversationScreenTopAppBarContent(
     )
 }
 
+@Composable
+private fun VerificationIcon(protocolInfo: Conversation.ProtocolInfo?) {
+    if (protocolInfo?.verificationStatus != Conversation.VerificationStatus.VERIFIED) return
```

Comment:
```suggestion
    if (protocolInfo?.verificationStatus == Conversation.VerificationStatus.NOT_VERIFIED) return
```

Reply:
do not agree. If the conversation status is Degraded we shouldn't display any icon just like it's not verified ([design](https://www.figma.com/file/gSmh7BRyq9SGKkyOLSbOdM/E2E-Identity?type=design&node-id=1-8174&mode=design&t=6EnAtJZfA9c1K9Ez-4) )